<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParallelProgram</class>
 <widget class="QMainWindow" name="ParallelProgram">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>390</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>740</width>
    <height>390</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>740</width>
    <height>390</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ParallelProgram</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSpinBox" name="logicThreadSpinBox">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>70</y>
      <width>441</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>128</number>
    </property>
   </widget>
   <widget class="QLabel" name="matrixGenerationLabel">
    <property name="geometry">
     <rect>
      <x>289</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Generated Time : </string>
    </property>
   </widget>
   <widget class="QLabel" name="logicThreadLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Logic Threads : </string>
    </property>
   </widget>
   <widget class="QSpinBox" name="maxElementsPerThreadSpinBox">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>40</y>
      <width>441</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100000000</number>
    </property>
   </widget>
   <widget class="QPushButton" name="startDynamicThreadsPushButton">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>210</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Dynamic Threads</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startSingleThreadPushButton">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>50</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Single Thread</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startTwoThreadsPushButton">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>90</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Two Threads</string>
    </property>
   </widget>
   <widget class="QPushButton" name="generatePushButton">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>10</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate</string>
    </property>
   </widget>
   <widget class="QLabel" name="matrixGenerationTimeLabel">
    <property name="geometry">
     <rect>
      <x>399</x>
      <y>10</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="matrixInfoLabel">
    <property name="geometry">
     <rect>
      <x>149</x>
      <y>10</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Matrix </string>
    </property>
   </widget>
   <widget class="QTableWidget" name="resultTableWidget">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>100</y>
      <width>581</width>
      <height>221</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <row>
     <property name="text">
      <string>Single</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Two</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Logic</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>2*Logic</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Dynamic</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Sum</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Threads</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>QTimer</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>OMPTime</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="startLogicThreadsPushButton">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>130</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Logic Threads</string>
    </property>
   </widget>
   <widget class="QLabel" name="maxElementsPerThreadLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Max Elements Per Thread : </string>
    </property>
   </widget>
   <widget class="QPushButton" name="start2xLogicThreadsPushButton">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>170</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Start 2*Logic Threads</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="generatedCheckBox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>10</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Generated</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>740</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
